packages/xdl/src/project/ExpSchema.ts

```diff
@@ -35,7 +36,8 @@ export async function validateAsync(projectRoot: string) {
 
 export async function getSchemaAsync(sdkVersion: string): Promise<Schema> {
```

Comment:
Every instantiation of a Schemer instance gets its schema from `getSchemaAsync` so we'll end up dereferencing the schemas twice. We should decide whether we want Schemer to require dereferenced schemas or if it will handle dereferencing.

Reply:
As there're other code paths that originate in `getSchemaAsync` method and do not go through the `Schemer` object I suggest having the dereferencing code in the `getSchemaAsync` only.